packages/ui5-application-writer/templates/optional/codeAssist/package.json

```diff
@@ -2,6 +2,6 @@
     "devDependencies": {
         "eslint": "7.32.0",
         "@sap/eslint-plugin-ui5-jsdocs": "2.0.5",
-        "@sapui5/ts-types": "1.102.x"
+        "@sapui5/ts-types": "<%- ui5.typesVersion %>"
```

Comment:
It is intentional to update codeAssist? 
I thought the types version calculation related to `@sapui5/ts-types-esm`

Reply:
Both are in sync and are released at the same time based on the same sources.